tests/helper/src/is_empty.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) The Libre Solar Project Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "helper.h"
+
+#include <stdio.h>
+
+#include <zephyr/ztest.h>
+
+ZTEST(is_empty, test_decreasing)
+{
+    float a[] = { 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 };
+    zassert_equal(1, is_empty((uint8_t *)a, sizeof(a)));
```

Comment:
Use of `zassert_true` and `zassert_false` would be more clear IMO:

```suggestion
    zassert_true(is_empty((uint8_t *)a, sizeof(a)));
```